### PR TITLE
:bomb: removes web.planner

### PR DESCRIPTION
**Pull request by michail93**

### CODE REVIEW

**Comment by ufaks** [comment]
@michail93 please remove js code about web_planner from here https://github.com/it-projects-llc/misc-addons/blob/master/web_debranding/static/src/js/dashboard.js#L26

**Comment by yelizariev** [comment]
@michail93  use :x: for removing

**Comment by yelizariev** [comment]
Also, fix your pull request — it has old commits

File: web_debranding/static/src/js/dashboard.js [hunk 1]
```diff
@@ -23,15 +23,6 @@ odoo.define('web_debranding.dashboard', function (require) {
                   <hr/>
                   Need more help? <a target="_blank" href="https://www.odoo.com/documentation/user">Browse the documentation.</a>
                 */
-                var $all = self.$('.o_web_settings_dashboard_planners_list').nextAll();
-                var text_node = $all[0].nextSibling;
```
Comment (yelizariev) [inline web_debranding/static/src/js/dashboard.js:27]: // Following code removes <hr/>, text node and <a/>
              


Which code?

**Pull request closed without merging**